quasiquotes3/shared/src/test/scala/scala/meta/tests/TestHelpers.scala

```diff
@@ -0,0 +1,21 @@
+package scala.meta.tests
+
+import scala.meta.Dialect
+// import scala.meta.Token
+
+object TestHelpers {
+
+  def getMessageWithExtraClue(msg: String, extraClue: String = ""): String =
+    if (extraClue.isEmpty) msg else s"$msg ($extraClue)"
+
+  def getSyntax(code: => String, syntax: String = null): String = Option(syntax).getOrElse(code)
+
+  // trait Tokenize {
```

Comment:
If not needed let's remove those

Reply:
Due to the current build structure, I had to copy some stuff from the tests module, including this (since I could not have it depend on the tests module, since it contains some macro implementations I believe). I could not copy everything (aforementioned macro and structure issues), so I commented out the problematic and unnecessary parts). I needed these files so as to not have to change the actual test file (for easier comparison between scala 2 and 3). What I am trying to say is that I think it will be better for me to polish this part later, when the build structure is set in stone (perhaps I will be able to completely get rid of this copy)